### PR TITLE
Feature/prep 0.7.0

### PR DESCRIPTION
**Pull request by DavidMinarsch**
## Proposed changes

This PR prepares develop for `0.7.0`. It bumps the versions of the aea and the packages.

## Fixes

-

## Types of changes

What types of changes does your code introduce to agents-aea?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply._

- [ ] I have read the [CONTRIBUTING](../CONTRIBUTING.md) doc
- [ ] I am making a pull request against the `develop` branch (left side). Also you should start your branch off our `develop`.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that code coverage does not decrease.
- [ ] I have checked that the documentation about the `aea cli` tool works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

-

### CODE REVIEW

File: aea/cli/remove.py [hunk 1]
```diff
@@ -125,13 +125,13 @@ def get_agent_dependencies_with_reverse_dependencies(
...
+            PackageId(protocol, fetchai/default:0.8.0): {
+                PackageId(skill, fetchai/echo:0.10.0),
+                PackageId(skill, fetchai/error:0.8.0)
             },
-            PackageId(connection, fetchai/stub:0.11.0): set(),
-            PackageId(skill, fetchai/error:0.7.0): set(),
-            PackageId(skill, fetchai/echo:0.9.0): set()}
+            PackageId(connection, fetchai/stub:0.12.0): set(),
+            PackageId(skill, fetchai/error:0.8.0): set(),
+            PackageId(skill, fetchai/echo:0.10.0): set()}
```
Comment (DavidMinarsch) [inline aea/cli/remove.py:134]: let's replace with arbitrary packages so no updating happens

File: aea/configurations/constants.py [hunk 2]
```diff
@@ -28,11 +28,11 @@
...
-DEFAULT_CONNECTION = PublicId.from_str("fetchai/stub:0.11.0")
-DEFAULT_PROTOCOL = PublicId.from_str("fetchai/default:0.7.0")
-SIGNING_PROTOCOL = PublicId.from_str("fetchai/signing:0.5.0")
-STATE_UPDATE_PROTOCOL = PublicId.from_str("fetchai/state_update:0.5.0")
-DEFAULT_SKILL = PublicId.from_str("fetchai/error:0.7.0")
+DEFAULT_CONNECTION = PublicId.from_str("fetchai/stub:0.12.0")
+DEFAULT_PROTOCOL = PublicId.from_str("fetchai/default:0.8.0")
+SIGNING_PROTOCOL = PublicId.from_str("fetchai/signing:0.6.0")
+STATE_UPDATE_PROTOCOL = PublicId.from_str("fetchai/state_update:0.6.0")
+DEFAULT_SKILL = PublicId.from_str("fetchai/error:0.8.0")
```
Comment (DavidMinarsch) [inline aea/configurations/constants.py:35]: let's point to latest, rather than specific versions!

Comment (marcofavorito) [reply aea/configurations/constants.py:35]: That might be a good idea, but I'd suggest doing it in a separate PR.

File: aea/cli/remove.py (hunk 1 above)
Comment (marcofavorito) [reply aea/cli/remove.py:134]: Agreed.

**Pull request merged**